sandpack-react/src/templates/react-typescript.ts

```diff
@@ -0,0 +1,86 @@
+import type { SandboxTemplate } from "../types";
+
+export const REACT_TYPESCRIPT_TEMPLATE: SandboxTemplate = {
+  files: {
+    "tsconfig.json": {
+      code: `{
+    "include": [
+        "./**/*"
+    ],
+    "compilerOptions": {
+        "strict": true,
+        "esModuleInterop": true,
+        "lib": [
+            "dom",
+            "es2015"
+        ],
+        "jsx": "react-jsx"
+    }
+}`,
+    },
+    "/App.tsx": {
+      code: `export default function App() {
```

Comment:
Could you type this component? It would be important for the user to understand they're using a different template.